schema-engine/connectors/mongodb-schema-connector/src/sampler/statistics.rs

```diff
@@ -147,6 +148,12 @@ impl<'a> Statistics<'a> {
                     None => renderer::datamodel::Model::new(name),
                 };
 
+                if let Some(walker) = doc_count.collection_walker {
+                    if walker.has_schema() {
+                        model.documentation("json schema msg")
+                    }
+                }
+
```

Comment:
This adds the documentation message that goes into the schema file above the model